website_sale_order_type/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website sale order type",
+    "summary": "This module allows sale_order_type to work with website_sale.",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Website",
+    "website": "https://github.com/OCA/e-commerce/",
+    "maintainers": ["joao-p-marques"],
+    "author": "Agile Business Group, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "website_sale",
+        "sale_order_type"
+    ],
+    "data": [
```

Comment:
```suggestion
    "demo": [
```

At least until v13, this improves performance.

Reply:
Thank you, changes made!